### PR TITLE
feat(helm-chart): Add extraLabels to all resources

### PR DESCRIPTION
**Pull request by maxforasteiro**
### SUMMARY
Some orgs require all Kubernetes resources to be labeled. Adding a non-breaking change that will allow our team to use Superset helm chart without having to copy/paste and modify it ourselves.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: helm/superset/values.yaml [hunk 1]
```diff
@@ -27,6 +27,11 @@ nameOverride: ~
 # -- Provide a name to override the full names of resources
 fullnameOverride: ~
 
+# -- Global values for the Superset chart
+global:
+  # -- Labels to be added to all resources
+  extraLabels: {}
+
```
Comment (villebro) [inline helm/superset/values.yaml:34]: Based on the comment here, these would be added to `all` resources. However, it seems we're not adding these to the pod templates (see for instance `.Values.supersetWorker.podLabels` - is this intentional?

**Review by villebro — COMMENTED** [review]
I like this idea. However, while I kind of like the `global` prefix, I'm wondering if it may be unnecessary. I believe we may already have some other global-like values that would seem non-global after this change. But I'm not super opinionated here.

Anyway, the main comment here applies to the labels not applying to the templates. Let's either apply them to those, too, or add a comment that they're not covered by this new property.

File: helm/superset/values.yaml (hunk 1 above)
Comment (maxforasteiro) [reply helm/superset/values.yaml:34]: sorry, I just missed them. I'm adding it now

**Comment by maxforasteiro** [comment]
> I like this idea. However, while I kind of like the `global` prefix, I'm wondering if it may be unnecessary. I believe we may already have some other global-like values that would seem non-global after this change. But I'm not super opinionated here.
> 
> Anyway, the main comment here applies to the labels not applying to the templates. Let's either apply them to those, too, or add a comment that they're not covered by this new property.

I agree with your comment, the global label can be misleading indeed.

**Review by villebro — APPROVED** [review]
LGTM, thanks for the improvement 🙇

**Comment by villebro** [comment]
Oh, please bump the version!

**Comment by maxforasteiro** [comment]
> Oh, please bump the version!

nice catch! completely forgot 🙈

**Comment by villebro** [comment]
Seems there's a linting issue:
image
I didn't look closely at it, let me know if you need help resolving it.

**Comment by maxforasteiro** [comment]
@villebro fixed. Sorry for the delay

**Review by villebro — APPROVED** [review]

**Pull request merged**